### ISSUE TITLE
Add a secret password to enable testing for darwin on windows and linux

### ISSUE THREAD

**Issue by Bluebugs** [fyne-io/fyne-cross#245]
### Is your feature request related to a problem? Please describe:

We currently can not tests target darwin from Linux and Windows host as it require a SDK that we can not distribute/share.

### Is it possible to construct a solution with the existing API?

No.

### Describe the solution you'd like to see:

Using Git LFS to store an encrypted version of MacOS SDK with the password stored in the secret. This is based on the idea described here for large secret: https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions . It would be good to disable Git LFS from Archive as described here: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-git-lfs-objects-in-archives-of-your-repository .

To summarize, I need a random password to be added to GitHub Actions secrets and communicate once to me. And it would be good to also disable Git LFS from Archive in the settings. Only the owner or an admin of the organization can do this change.

**Comment by calderonth** [comment]
I would be happy to store this in an S3 compatible bucket for you guys and provide an access secret key that you can inject as part of your CI.
